src/components/image-content-block/image-content-block.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { Trans } from 'react-i18next';
+import styled, { css } from 'styled-components';
+import { up } from '../breakpoint/breakpoint';
+import { theme } from '../layout/theme';
+import { TextStyles } from '../typography/typography-v2';
+
+const mapTextAlignToFlex = (textAlign: 'right' | 'left' | 'bottom') => {
+  if (textAlign === 'right') {
+    return 'row';
+  } else if (textAlign === 'left') {
+    return 'row-reverse';
+  } else {
+    return 'column';
+  }
+};
+
+interface TextAlignProps {
+  textAlign: 'right' | 'left' | 'bottom';
+}
+
+const BlockWrapper = styled.div<TextAlignProps>`
+  display: inline-flex;
```

Comment:
Why would you define this an an inline container? Just make it flex which renders as a block to spread to the full width. That way you can also remove the width 100%.

Reply:
Way cleaner, thank you 👍